lib/tests/test_mutations.c

```diff
@@ -78,13 +78,13 @@ insert_single_tree(tsk_table_collection_t *tables, int alphabet)
         ret = tsk_site_table_add_row(&tables->sites, 0.1, "0", 1, NULL, 0);
         CU_ASSERT_FATAL(ret >= 0);
         ret = tsk_mutation_table_add_row(
-            &tables->mutations, 0, 0, -1, TSK_UNKNOWN_TIME, "1", 1, NULL, 0);
+            &tables->mutations, 0, 0, -1, 0.0, "1", 1, NULL, 0);
```

Comment:
we should still have unknown times in the tests, btw.

Reply:
why?

Reply:
We will still need to add mutations to tables with unknown mutation times.

Reply:
(I think we decided we could only do that with not-discrete mutation positions, though.)

Reply:
given the below, I guess we only need to make sure that unknown-ness is preserved at pre-existing mutations that aren't at newly mutated sites

Reply:
done